direct_db_manager.py

```
@@ -12,7 +13,11 @@
 import sqlite3
 import json
 import hashlib
+from PyQt5.QtCore import QThread, pyqtSignal
 import resources
+import queue
+import weakref
+from PyQt5.QtCore import QMutex, QWaitCondition, QPropertyAnimation, QEasingCurve
```

Ruff (0.12.2)

`PyQt5.QtCore.QPropertyAnimation` imported but unused

Remove unused import

(F401)